### PR TITLE
Initial Framework complete

### PR DESCRIPTION
**Pull request by beazley2**
## What is done
- Guide Model added with single location field
- DB seeding updated. 99 users, 40 of which are upgraded to guides
- localhost:3000/guides/show has a listing of all guides currently in DB:
guides

## What still needs to be done
- Remaining fields need to be added to both users and guides. I will continue working on this after lunch today.
- Webpage for guide/new.html.erb not done. This means you can't added new guides via the web yet. If someone on the backend team wants to take a crack at it, use user/new.html.erb as a guide

### CODE REVIEW

File: app/models/guide.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+class Guide < ActiveRecord::Base
+  belongs_to :user
+
+  #validates :user_id, presence: true
```
Comment (beazley2) [inline app/models/guide.rb]: This should not be commented out; did while trying to debug the test suite. Already fixed locally.

**Comment by ardeshirj** [comment]
Ticket #11 
Great job brad. I will check to see if I can help with the known issues.

**Comment by ardeshirj** [comment]
Lets update db:seed to have insert some random availability.

**Comment by beazley2** [comment]
It already does, lines 59-70, unless you are talking specific times.

**Comment by lewisric** [comment]
Merged into master and bot might work...

**Pull request merged**